pkgs/games/n2048/default.nix

```diff
@@ -1,35 +1,26 @@
 {lib, stdenv, fetchurl, ncurses}:
-let
-  s =
-  rec {
-    baseName = "n2048";
+
+stdenv.mkDerivation rec {
+    pname = "n2048";
     version = "0.1";
```

Comment:
nit: this leaves odd indentation in the new version

Reply:
sorry I forgot to run `nixpkgs-fmt`. should be fixed now